ci/windows/test_py3_cpu.ps1

```diff
@@ -24,7 +24,7 @@ $env:MXNET_HOME=[io.path]::combine($PSScriptRoot, 'mxnet_home')
 
 C:\Python37\Scripts\pip install -r tests\requirements.txt
 C:\Python37\python.exe -m nose -v --with-timer --timer-ok 1 --timer-warning 15 --timer-filter warning,error --with-xunit --xunit-file nosetests_unittest.xml tests\python\unittest
-if (! $?) { Throw ("Error running unittest") }
+if (! $?) { Throw ("Error running unittest) }
```

Comment:
Are you sure you want to remove the `"`?

Reply:
whoops, I was factoring some error-handling code out to PR-15147 and got a little too aggressive.  Good catch, fixed now.